src/task/models/tasksManager.ts

```diff
@@ -0,0 +1,109 @@
+import { Logger } from '@map-colonies/js-logger';
+import { CreateExportTaskRequest, IExportManager, TaskParameters } from '@map-colonies/export-interfaces';
+import { inject, injectable } from 'tsyringe';
+import { Domain } from '@map-colonies/types';
+import { BadRequestError, NotFoundError } from '@map-colonies/error-types';
+import { SERVICES } from '../../common/constants';
+import { ExportManagerRaster } from '../../exportManager/exportManagerRaster';
+import { TASK_REPOSITORY_SYMBOL, TaskRepository } from '../../DAL/repositories/taskRepository';
+import { TaskEntity } from '../../DAL/entity/task';
+import { ITaskEntity } from '../../DAL/models/task';
+
+@injectable()
+export class TasksManager {
+  public constructor(
+    @inject(SERVICES.LOGGER) private readonly logger: Logger,
+    @inject(TASK_REPOSITORY_SYMBOL) private readonly taskRepository: TaskRepository
+  ) {}
+
+  public async createTask(req: CreateExportTaskRequest<TaskParameters>): Promise<ITaskEntity> {
+    try {
+      this.logger.info({ msg: `Creating export task`, req: req });
+
+      const domain = req.domain;
+      const exportManagerInstance = this.getExportManagerInstance(domain);
+      // TODO: Call Domain SDK
+      const exportTaskResponse = await exportManagerInstance.createExportTask(req);
+      const jobId = exportTaskResponse.jobId;
+      const entity = await this.taskRepository.getTaskById({ jobId });
+      // return the entity if its already exists
+      if (entity) {
+        this.logger.debug(entity, `Found an entity with the same job ib: ${jobId}`);
+        return entity;
+      }
+      // TODO Call Domain SDK to get estimations
+      const exstimations = await exportManagerInstance.getEstimations();
+      // TODO: Get customer name
+      const customerName = 'Cutomer_Name';
+      const newEntity = new TaskEntity();
+      Object.assign(newEntity, req, {
+        taskGeometries: exportTaskResponse.geometries,
+        jobId: exportTaskResponse.jobId,
+        customerName,
+        estimatedDataSize: exstimations.estimatedFileSize,
+        estimatedTime: exstimations.estimatedTime,
+      });
+
+      const res = await this.taskRepository.createTask(newEntity);
+      return res;
```

Comment:
add log that task was created with it's main params

Reply:
add also an info log with main params (export task Id, jobId, domain, customer, catalogRecordID)